base/sparse/sparsematrix.jl

```diff
@@ -2954,21 +2954,56 @@ function is_hermsym(A::SparseMatrixCSC, check::Func)
     rowval = A.rowval
     nzval = A.nzval
     tracker = copy(A.colptr)
-    @inbounds for col = 1:A.n
+    for col = 1:A.n
+        # By how `tracker` is updated,, for symmetric matrices the loop below
```

Comment:
lots of typos in comments - how about "`tracker` is updated such that, for symmetric matrices, the loop below starts from an element at or below the diagonal element of column `col`"